teststack.cpp

```diff
@@ -1,7 +1,28 @@
 #include "problems.h"
 using namespace std;
 
+
+
 int main()
 {
+
```

Comment:
I had my own tests to confirm your code but I do really think you should think about adding more of your own when you do these problem sets!

50/50 - Nice work!